src/lib/elements/forms/inputFile.svelte

```diff
@@ -47,6 +36,11 @@
         hovering = false;
         if (!ev.dataTransfer.items) return;
         for (let i = 0; i < ev.dataTransfer.items.length; i++) {
+            const fileExtension = ev.dataTransfer.items[i].getAsFile().name.split('.')[1];
+            if (!allowedFileExtensions.includes(fileExtension)) {
```

Comment:
If `allowedFileExtensions` is an empty array, all file extensions should be allowed.

Reply:
Noted